gocli/cmd/run.go

```diff
@@ -328,6 +374,22 @@ func run(cmd *cobra.Command, args []string) (err error) {
 		}(node.ID)
 	}
 
+	// If logging is enabled, deploy the default fluent logging
+	if logDir != "" {
+		nodeName := nodeNameFromIndex(1)
+		success, err := docker.Exec(cli, nodeContainer(prefix, nodeName), []string{
+			"/bin/bash",
+			"-c",
+			"ssh.sh sudo /bin/bash < /scripts/logging.sh",
```

Comment:
Ok, if we do it this way, we would not have precashed fluentd containers in the image. So far we tried to have everything ready to start. Maybe you could put the manifests in the same directlry like  @awels in #26.